VERSION

```diff
@@ -1 +1 @@
-0.18.0
+0.18.1
```

Comment:
This needs to get pulled out we don't do version bumps for individual changes at the moment.